### PR TITLE
support query endpoint list with time range

### PR DESCRIPTION
**Pull request by mrproliu**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
fyi @apache/skywalking-committers We are open endpoint keep alive due to r3 is stable and restful APIs are not our concern to cause high cardinal number issues.

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/endpoint/EndpointTraffic.java [hunk 1]
```diff
@@ -48,6 +48,7 @@ public class EndpointTraffic extends Metrics {
 
     public static final String SERVICE_ID = "service_id";
     public static final String NAME = "endpoint_traffic_name";
+    public static final String LAST_PING_TIME_BUCKET = "last_ping";
```
Comment (wu-sheng) [inline oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/endpoint/EndpointTraffic.java:51]: I think endpoint traffic declared no update.

Could you verify the data in database?

Comment (mrproliu) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/endpoint/EndpointTraffic.java:51]: I have checked in Elasticsearch storage, looks like the update successful.
image

Comment (wu-sheng) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/endpoint/EndpointTraffic.java:51]: Check here, 

MetricsExtension(supportDownSampling = false, supportUpdate = false)

We declared this metric doesn't support update. What happens?

Comment (wu-sheng) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/endpoint/EndpointTraffic.java:51]: https://github.com/apache/skywalking/blob/29618d7f2e795f1822d46015c23edc3f6b0c66be/oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/worker/MetricsPersistentWorker.java#L252-L257

According to this, an existing metric(if no_update=true) should not re-entry the `prepareRequests` list, and end in database again.

File: docs/en/changes/changes.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
 * Reduce the count of process index and adding time range when query process index.
 * Bump up Apache commons-io to 2.17.0.
 * Polish eBPF so11y metrics and add error count for query metrics.
+* Support query endpoint list with duration parameter.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:10]:
```suggestion
* Support query endpoint list with duration parameter(optional).
* Change the endpoint_traffic to updatable for the additional column `last_ping`.
```

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**